frontend/app/ui_components/has-dropdown-menu-directive.js

```diff
@@ -70,22 +102,18 @@ module.exports = function($injector, $window, $parse) {
     link: function(scope, element, attrs, ctrl) {
       var contextMenu = $injector.get(attrs.target),
         locals = {},
+        pointerPosition,
+        pointerCssPosition,
         win = angular.element($window),
         menuElement,
+        afterFocusOn = attrs.afterFocusOn
         positionRelativeTo = attrs.positionRelativeTo,
-        triggerOnEvent = attrs.triggerOnEvent || 'click';
+        triggerOnEvent = (attrs.triggerOnEvent || 'click') + '.dropdown.openproject';
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>'triggerOnEvent' is not defined.